.eslintrc.yaml

```diff
@@ -163,3 +149,78 @@ rules:
     '@typescript-eslint/explicit-function-return-type': [error, {
       allowIIFEs: true, # IIFEs are widely used, writing their signature twice is painful
     }]
+
+
+    # constructor-super: error
+    # curly: [ error, all ]
+    # no-debugger: error
+    # no-duplicate-case: error
+    # no-eval: error
+    # no-sparse-arrays: error
+    # no-trailing-spaces: error
+    # no-var: error
+    # prefer-const: error
+    # prefer-numeric-literals: off
+    # prefer-template: error
+
+
+    # '@typescript-eslint/no-unsafe-argument': off
+
+    max-len: [warn, 250] # more lenient on max length per line
+    no-console: warn # prefer the uniform logging methods
+    '@typescript-eslint/no-empty-function': off
+    no-multi-spaces: [warn, {'ignoreEOLComments': true, 'exceptions': {'Property': true, 'VariableDeclarator': false } }]
+
+    comma-dangle: [error, {
```

Comment:
Why don't you choose to disallow trailing commas? I think it'd better to allow. For example, if you had the following array:

```ts
const array = [
  a,
  b
];
```

In this time, you want to append a new item, the diff might be look like:

```diff
- b
+b,
+c
```

In compare with the allowing case, the diff would be:

```diff
  b,
+c,
```

Reply:
I've changed the config. I think you are right.